agent/engine/docker_task_engine_test.go

```diff
@@ -156,6 +156,7 @@ func TestBatchContainerHappyPath(t *testing.T) {
 	default:
 	}
 	eventsReported.Wait()
+	client.EXPECT().DescribeContainer(gomock.Any()).AnyTimes()
```

Comment:
This comments seems wrong. 
In docker_task_engine.go code, there are ONLY 2 possible "Callstacks" for this 'DescribeContainer()'.
One from CheckTaskState() <- overseeTask() <- startTask()
One from synchronizeState()<-Init().   With the test data "sleep5.json" used in this testcase, there is ONLY one possible "Callstack" (overseeTask).